lib/db.js

```diff
@@ -0,0 +1,37 @@
+var matchPattern = require('lodash-match-pattern');
+var expect = require('chai').expect;
+
+module.exports = {
+  truncateTables: function () {
```

Comment:
@allonsy -- the main role of this one is to be in the before hooks.  We'll come back to it shortly.